src/Main.ps1

```diff
@@ -921,6 +922,11 @@ function Invoke-Pester {
                 $PesterPreference.Debug.WriteDebugMessagesFrom = $PesterPreference.Debug.WriteDebugMessagesFrom.Value + @($missingCategories)
             }
 
+            if ($PesterPreference.Debug.ShowFullErrors.Value) {
```

Comment:
Not sure if this a great place to put the warning message.

Reply:
Just add a note to the config object that this is deprecated and setting it to true will instead set <option in output> to Full. And then do that in code. No need to write to user screen for such minor option 🙂 It's probably just us using it anyway.

Reply:
> Not sure if I should go through every call to Debug.ShowFullErrors and replace it with the new option?

You don't have to do it. It can be done when the option is removed somewhere in the future.

Reply:
I've updated the configuration descriptions and removed the warning.